### ISSUE TITLE
"Random is not defined"

### ISSUE THREAD

**Issue by SolasCo** [musicalglass/KA_Offline#1]
Any ideas as to why when I paste my code in and try to run it, it says "Random is not defined".
This is all I am using. 
var generator = new Random(1);

**Comment by musicalglass** [comment]
because random does not use a Capital R?


________________________________

**Comment by trn1ty** [comment]
Yep, functions are caps-sensitive. random() is different from Random() according to JS.
